foxbroadcasting/react-select
# [DPP-1339] Change chevron to arrow if unique is not selected

## What did you implement:

Implementing issue: [DPP-1339](https://fng-jira.fox.com/browse/DPP-1339)

## How did you implement it:
Added a validation to `defaultArrowRenderer` to see if it's unique or not, in order to render the chevrown svg or a span with the `Select-arrow` class.

## How can we verify it:
1. In the DPP repo fng-cms change branch to feature/DPP-1339
2. On the DPP package.json file replace the @foxdcg/react-select dependency with this:
```
"@foxdcg/react-select": "git://github.com/foxbroadcasting/react-select.git#bug/DPP-1339",
```
3. `Run npm i && npm start`
4. Open the DPP on the browser
5. Go to http://local.dpp.foxdcg.com:3000/new/videolists?filter=
6. See the difference between arrow and chevron in the react-selects.

## Screenshots
Before
<img width="918" alt="screen shot 2019-01-10 at 12 31 12 pm" src="https://user-images.githubusercontent.com/41165303/51002031-a3420000-14f7-11e9-9c70-d1027aeb8535.png">


After
<img width="923" alt="screen shot 2019-01-10 at 4 49 14 pm" src="https://user-images.githubusercontent.com/41165303/51002043-af2dc200-14f7-11e9-89c9-0e14362243b2.png">





## Related PRs
This PR Should be merged before [DPP-1321](https://fng-jira.fox.com/browse/DPP-1321) in order to function correctly according to the UX Mocks. 

## Checklist
